docker-compose.yml

```diff
@@ -318,92 +301,84 @@ services:
     ports:
       - "3002:3002"
 
-  squid-archive:
-    container_name: squid-archive
```

Comment:
Wouldn't make more sense for a playground's default deployment to have its own squid-archive and gateway and configure default value of `ARCHIVE_GATEWAY_URL` to point to it?

Reply:
ok